src/common/read-maybe-existed-file.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
```

Comment:
I have been moving these kind of utility files to `utils/`, since `common/` was getting too crowded but I'm not sure that's making a real difference 😂